Code/Editor/EditorFramework/EditorApp/EditorApp.cpp

```diff
@@ -367,7 +367,11 @@ ezStatus ezQtEditorApp::MakeRemoteProjectLocal(ezStringBuilder& inout_sFilePath)
 
     QProcess proc;
     proc.setWorkingDirectory(sTargetDir.GetData());
+#if EZ_ENABLED(EZ_PLATFORM_WINDOWS_DESKTOP)
```

Comment:
We have similar code elsewhere, not sure whether or does the same thing, it whether it appends a platform specific suffix or calls some function to do that.

Reply:
So I've looked around and we have multiple cases of the platform ifdef pattern like this one here, but there is also this pattern:
```
#  if EZ_ENABLED(EZ_PLATFORM_WINDOWS)
  pathToArchiveTool.Append(".exe");
#  endif
```
And a lot of: "ups we forgot to make this work on linux"!
Haven't found anything more fancy than those.